.github/workflows/kong3.0_compatibility.yaml

```diff
@@ -75,11 +75,14 @@ jobs:
       id: run_integration_tests
       run: make _test.integration
       continue-on-error: true
+      ## skip enterprise-dbless option.
+      if: ${{ matrix.kong-edition == 'OSS' || matrix.database-mode == 'postgres' }} 
       env:
         TEST_KONG_IMAGE: ${{ env.TEST_KONG_IMAGE }}
         TEST_KONG_TAG: ${{ env.TEST_KONG_TAG }}
         TEST_KONG_ENTERPRISE: ${{ env.TEST_KONG_ENTERPRISE }}
         KONG_LICENSE_DATA: ${{ steps.license.outputs.license }}
+        DBMODE: ${{ env.TEST_DATABASE_MODE }}
         TEST_DATABASE_MODE: ${{ env.TEST_DATABASE_MODE }}
```

Comment:
Why use 2 variables with the same value? Can't we change the code to use just one of those?

Reply:
BTW: if we change the values in the matrix from `dbless` to `off` I think we could then use just that instead of assigning it like so (above)

```
      if: ${{ matrix.database-mode == 'dbless' }}
      run: echo "TEST_DATABASE_MODE=off" >> $GITHUB_ENV
```

If we do the change then it's enough to 

```
        TEST_DATABASE_MODE: ${{ matrix.database-mode }}
```

right?

Reply:
attempted solving in 78a92469 - seems that TEST_DATABASE_MODE is set by make targets internally based on the incoming value of DBMODE. So it feels wrong to set TEST_DATABASE_MODE when calling `make`. Let's see if this works :crossed_fingers:

Reply:
fixed it. And found one more env `NCPU` that might make dbless test to fail.